src/GraphQl/Action/EntrypointAction.php

```diff
@@ -94,21 +95,17 @@ private function parseRequest(Request $request): array
         }
 
         if ('json' === $request->getContentType()) {
-            $input = json_decode($request->getContent(), true);
-
-            if (isset($input['query'])) {
-                $query = $input['query'];
-            }
+            return $this->parseInput($query, $variables, $operation, $request->getContent());
+        }
 
-            if (isset($input['variables'])) {
-                $variables = \is_array($input['variables']) ? $input['variables'] : json_decode($input['variables'], true);
-            }
+        if (false !== mb_stripos($request->headers->get('CONTENT_TYPE'), 'multipart/form-data')) {
```

Comment:
Why using the multi-bytes version? And why not use `===`?

Reply:
About the multi-bytes because content type header has boundary something like this:
`multipart/form-data; boundary=----WebKitFormBoundaryPxBqLqriGaLFEYzV`
which can be set by user, I thought maybe some user send UTF-8 data in there and it may cause a problem.
about the `===` I used it. its just mot equal version of it: `!==`